parts/k8s/kubernetesconfigs.sh

```diff
@@ -420,5 +437,4 @@ configGPUDrivers() {
 ensureGPUDrivers() {
     configGPUDrivers
     systemctlEnableAndStart nvidia-modprobe || exit $ERR_GPU_DRIVERS_START_FAIL
-    retrycmd_if_failure 30 1 120 systemctl restart kubelet || exit $ERR_GPU_DRIVERS_START_FAIL
```

Comment:
why remove this?

Reply:
the k8s runtime hasn't been commenced yet, I think this is a no op (or actually a bug)

Reply:
there is a comparable restart later in the CSE flow